# HG changeset patch
# User msirringhaus@suse.de
# Date 1559717288 -7200
#      Wed Jun 05 08:48:08 2019 +0200
# Node ID 42c99b59a87b904063bad3193f10c51d068d2eac
# Parent  386083b58d8558141901d796ec6919a4aba7ad3a
Fix compile error with inline assembler for s390x and ppc64le.

diff -r 386083b58d85 -r 42c99b59a87b security/nss/lib/freebl/mpi/mpcpucache.c
--- a/security/nss/lib/freebl/mpi/mpcpucache.c	Tue May 21 14:49:58 2019 +0200
+++ b/security/nss/lib/freebl/mpi/mpcpucache.c	Wed Jun 05 08:48:08 2019 +0200
@@ -704,7 +704,7 @@
 #define MPI_GET_PROCESSOR_LINE_SIZE_DEFINED 1
 #endif
 
-#if defined(__ppc64__)
+#if 0
 /*
  *  Sigh, The PPC has some really nice features to help us determine cache
  *  size, since it had lots of direct control functions to do so. The POWER
